app/controllers/accounts_controller.rb

```diff
@@ -78,6 +78,14 @@ def destroy
 
   private
 
+    def decide_layout
+      if can? :read, :admin_dashboard
```

Comment:
See the blue button: 
<img width="1344" alt="screen shot 2016-09-01 at 5 06 44 pm" src="https://cloud.githubusercontent.com/assets/92044/18186116/7d11d1fe-7066-11e6-9069-9d847c516e9a.png">